### ISSUE TITLE
Duplicate enum key in executable/elf.ksy

### ISSUE THREAD

**Issue by generalmimon** [kaitai-io/kaitai_struct_formats#279]
The [ELF format](https://github.com/kaitai-io/kaitai_struct_formats/blob/2c318560d46a64a3338e531ff1540716dab9798c/executable/elf.ksy) no longer compiles with the modern 0.9 compiler, because it contains a duplicate `enum` key:

https://github.com/kaitai-io/kaitai_struct_formats/blob/2c318560d46a64a3338e531ff1540716dab9798c/executable/elf.ksy#L604-L605

Here's the error:
```
io.kaitai.struct.precompile.YAMLParserError: D:/temp/kaitai_struct/formats/executable/elf.ksy:605:5: found duplicate key 1879048193
```

The YAML parser used in the stable Web IDE (https://ide.kaitai.io/) silently uses the first encountered value, I haven't tried the desktop one. But it's clear that this is a bug in the format that needs to be fixed.

**Comment by taviso** [comment]
I tried this fix, and it still doesn't compile - I think because of these two dynamic tags:

```
    32: encoding         # Start of encoded range
    32: preinit_array    # Array with addresses of preinit fct
```

I think it's safe to delete the encoding one, IIUC, it's just used for comparing in tools and not encoded in the file:  https://docs.oracle.com/cd/E19683-01/816-7529/chapter6-42444/index.html

**Comment by taviso** [comment]
OK, I fixed them here #288
